packages/asset/contracts/interfaces/IExchange.sol

```diff
@@ -1,9 +1,8 @@
 // SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
```

Comment:
I think interfaces should be floating pragma when possible, so others packages with different pragma can still use it

Reply:
still fixed version

Reply:
@mvanmeerbeck sorry, fixed this one, I've change asset create interface to floating pragma instead before